### ISSUE TITLE
Change page `<title>`s of support admin pages

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#5182]
## Area or Component
Support admin pages:
- /admin
- /admin/pending-organizations
- /admin/add-organization-admin
- /admin/create-device-type
- /admin/manage-devices
- /admin/tenant-data-access

## Quick Summary
The support admin pages share the same page `<title>` of "SimpleReport"

## Accessibility Violation
[Page title does not identify purpose of page](https://cdc-devtools.dequecloud.com/rule-help/en/title-not-meaningful)

## Action Requested
Update the `<title>` of the support admin pages so they are unique and identifies the purpose of the page

## Source Code (optional)

[which file or line is the issue attached to?]: #
